rollup/conf/config.json

```diff
@@ -59,7 +58,8 @@
         "try_times": 5,
         "base_url": "http://localhost:8750"
       },
-      "finalize_batch_interval_sec": 0,
+      "is_test_env": true,
```

Comment:
How about change it to `environment=mainnet|sepolia|staging`, and a common method check environment

Reply:
If so, Sepolia would enable "timeout finalize" by default.